<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>206</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="instrumentationOptions">
      <attribute name="title">
       <string>Instrumentation</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="instrumentSettings">
         <property name="title">
          <string>Instrumentation Type</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Polarity / Ionization Mode:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="ionizationMode">
            <item>
             <property name="text">
              <string>Auto Detect</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Neutral (0)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Positive (+1)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Negative (-1)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Q1 Accuracy</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="amuQ1">
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string> amu</string>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="ionizationType">
            <item>
             <property name="text">
              <string>ESI</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EI</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CI</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>APCI</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FI/FD</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FAB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MALDI</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Ionization Type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Q3 Accuracy</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="amuQ3">
            <property name="suffix">
             <string> amu</string>
            </property>
            <property name="maximum">
             <double>1000000000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Instrument Type:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="instrumentType">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Full Spectrum</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Triple Quad</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>File Import</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="3" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Scan Filter Intensity Minimum Quantile Cutoff </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="centroid_scan_flag">
         <property name="text">
          <string>Centroid Scans</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="scan_filter_min_quantile">
         <property name="suffix">
          <string> %</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="scan_filter_min_intensity">
         <property name="suffix">
          <string> ions</string>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="maximum">
          <number>1000000000</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Scan Filter:  Minimum Intensity </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="scan_filter_polarity">
         <item>
          <property name="text">
           <string>Not Filtered</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Positive Polarity Scans Only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Negative Polarity Scans Only</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Scan Filter Polarity</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Scan Filter MS Level</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="scan_filter_mslevel">
         <item>
          <property name="text">
           <string>Not Filtered</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MS1 Scans Only</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MS2 Scans Only</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="peakDetectOptions">
      <attribute name="title">
       <string>Peak Detection</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Peak Detection and Grouping Settings</string>
         </property>
         <layout class="QGridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>EIC Smoothing Window </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="eic_smoothingWindow">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>scans</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Max Retention Time difference between Peaks</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>min</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="grouping_maxRtWindow">
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.250000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>EIC Smoothing Algorithm</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="eic_smoothingAlgorithm">
            <item>
             <property name="text">
              <string>Savitzky-Golay</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gaussian</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Moving Average</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="recomputeEICButton">
         <property name="text">
          <string>Recompute EIC</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Baseline Calculation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Baseline Smooting </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="baseline_smoothing">
            <property name="suffix">
             <string> scans</string>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="baseline_quantile">
            <property name="suffix">
             <string> %</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>80</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Drop top x%  intensities from chromatogram</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="isotopeDetectOptions">
      <attribute name="title">
       <string>Isotope Detection</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Samples Are Labeled? </string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QCheckBox" name="C13Labeled">
            <property name="text">
             <string>C13 Labeled </string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="N15Labeled">
            <property name="text">
             <string>N15  Labeled</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="S34Labeled">
            <property name="text">
             <string>S34 Labeled</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="D2Labeled">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>D2 Labeled</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Filter Isotopic Peaks based on these criteria</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Maximum %Error to Natural Abundance</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="maxNaturalAbundanceErr">
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string> %</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="value">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Isotope is within [X] scans of parent</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Minimum Isotope-parent correlation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="minIsotopicCorrelation">
            <property name="minimum">
             <double>-1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.200000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="maxIsotopeScanDiff">
            <property name="suffix">
             <string> scans</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="isotopeC13Correction">
            <property name="text">
             <string>Correct for Natural C13 Isotope Abundance</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Web Servers</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Location of Remote and Local Libraries</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Remote Data Server</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="data_server_url">
            <property name="text">
             <string>http://genomics-pubs.princeton.edu/mzroll/</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Methods Folder</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="methodsFolderSelect">
            <property name="icon">
             <iconset resource="../mzroll.qrc">
              <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="methodsFolder"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Pathways Folder</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pathwaysFolderSelect">
            <property name="icon">
             <iconset resource="../mzroll.qrc">
              <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="pathwaysFolder"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Scripts Folder</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="scriptsFolderSelect">
            <property name="icon">
             <iconset resource="../mzroll.qrc">
              <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="scriptsFolder"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>External Programs</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>R Executable </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="RProgramSelect">
         <property name="icon">
          <iconset resource="../mzroll.qrc">
           <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="Rprogram"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mzroll.qrc"/>
 </resources>
 <connections/>
</ui>
